task/source-build/0.1/source-build.yaml

```diff
@@ -157,5 +159,6 @@ spec:
 
         cat "$RESULT_FILE" | jq -j ".image_url" >"$RESULT_SOURCE_IMAGE_URL"
         cat "$RESULT_FILE" | jq -j ".image_digest" >"$RESULT_SOURCE_IMAGE_DIGEST"
+        jq -r '"\(.image_url)@\(.image_digest)"' "${RESULT_FILE}" >"$(results.IMAGE_REF.path)"
```

Comment:
This will include a newline at the end (I think the previous jq calls use `-j` to avoid that)